Cargo.toml

```diff
@@ -204,6 +204,7 @@ generic-array = { version = "0.14.7", default-features = false }
 gethostname = "0.2.3"
 getrandom = "0.2.10"
 goauth = "0.13.1"
+hdrhistogram = "7.5.2"
```

Comment:
There's already a histogram crate, what is the difference/benefit of adding another one?

Reply:
> Should I look into bumping our usage of the histogram crate from v0.6 to v0.7?

Seems like it should be separate from this PR, but if it has features that improve the interface, I vote yes 😄